controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -448,6 +458,46 @@ bool master_thread::autoconfig_wsc_calculate_keys(std::shared_ptr<ieee1905_1::tl
     return true;
 }
 
+/**
+ * @brief autoconfig global authenticator attribute calculation
+ * 
+ * Calculate authentication on the Full M1 || M2* whereas M2* = M2 without the authenticator
+ * attribute.
+ * 
+ * @param m1 WSC M1 TLV
+ * @param m2 WSC M2 TLV
+ * @param authkey authentication key
+ * @return true on success
+ * @return false on failure
```

Comment:
Does doxygen allow two @return tags?

Reply:
This AutoGenerated by vscode doxygen extension so I'd assume it does, but I never actually generated doxygen so not sure.